### PR TITLE
Updating sql files - Closes #1453

### LINKED ISSUE #1453: Update SQL files
As per the changes agreed earlier, to follow our standards:

* Rename camel-case names into underscore, to be consistent with everything else
* Add the Copyright notice into each SQL file
* Add the SQL-related standards into [db/sql/README.md](https://github.com/LiskHQ/lisk/tree/development/db/sql)

### PR DESCRIPTION
**Pull request by vitaly-t**
Closes #1453

---

* Renames all SQL files into underscore case, including the migration files in `db/sql/migrations/updates`
* Adds Copyright note and a description block into each SQL file
* Updates file references for all SQL files
* Updates regular expression for migration files, to allow anything in the name after `12345_`, including letters, digits and underscores.
* Updates `db/sql/README.md` with all the instructions regarding SQL files

### CODE REVIEW

File: db/sql/README.md [hunk 1]
```diff
@@ -4,4 +4,48 @@ sql files
 This folder contains all SQL files used in the project:
 
 * A folder for each database repository, containing SQL files used by that repository.
-* File [index](./index.js) references and loads all SQL files in the project.
+* File [./config.js](./config.js) provides the tools for loading external SQL files.
+* File [./index.js](./index.js) individually references and loads each SQL file in the project.
+
+## standards
```
Comment (willclarktech) [inline db/sql/README.md]: Prefer capitals for headings. (See other .md files.)

File: db/sql/accounts/column_delegates.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/*
```
Comment (willclarktech) [inline db/sql/accounts/column_delegates.sql]: In general, prefer `git mv` so we can see can track which files have been moved where.

Comment (vitaly-t) [reply db/sql/accounts/column_delegates.sql]: I moved files visually in Webstorm IDE, presuming it knows what it is doing underneath :)

File: db/sql/README.md (hunk 1 above)
Comment (vitaly-t) [reply db/sql/README.md]: I will keep improving this document in future PR-s, and surely will accommodate this case change :wink:

File: db/sql/accounts/column_delegates.sql (hunk 2 above)
Comment (willclarktech) [reply db/sql/accounts/column_delegates.sql]: Actually I may have misunderstood how `git mv` works: https://git.wiki.kernel.org/index.php/GitFaq#Why_does_Git_not_.22track.22_renames.3F

Comment (vitaly-t) [reply db/sql/accounts/column_delegates.sql]: :smile: I was expecting Webstorm to be doing nothing short of optimal git commands for that :)

**Comment by karmacoma** [comment]
@vitaly-t @nazarhussain We will merge this PR and deal with inconsistencies afterwards, as expressed by @4miners in #1457.

File: db/sql/README.md (hunk 1 above)
Comment (vitaly-t) [reply db/sql/README.md]: @willclarktech this change has been applied and merged ;)

**Comment by vitaly-t** [comment]
@nazarhussain I did [PR-1473](https://github.com/LiskHQ/lisk/pull/1473) that took care of it :wink:

File: db/sql/index.js [hunk 3]
```diff
@@ -17,87 +17,87 @@ const {link} = require('./config');
...
-		performRoundSnapshot: link('rounds/performRoundSnapshot.sql'),
-		restoreRoundSnapshot: link('rounds/restoreRoundSnapshot.sql'),
-		clearVotesSnapshot: link('rounds/clearVotesSnapshot.sql'),
-		performVotesSnapshot: link('rounds/performVotesSnapshot.sql'),
-		restoreVotesSnapshot: link('rounds/restoreVotesSnapshot.sql'),
-		getMemRounds: link('rounds/getMemRounds.sql'),
+		getVotes: link('rounds/get_votes.sql'),
+		updateVotes: link('rounds/update_votes.sql'),
+		updateMissedBlocks: link('rounds/update_missed_blocks.sql'),
+		updateBlockId: link('rounds/update_blockId.sql'),
```
Comment (vitaly-t) [inline db/sql/index.js]: Somehow missed this one, and the corresponding file at the same time! :smile:

**Pull request merged**